content/games/data.toml

```diff
@@ -157,6 +157,14 @@ categories = ["simulation"]
 repository_url = "//github.com/amethyst/evoli/"
 image = "/assets/img/evoli-prealpha.gif"
 
+[[items]]
+name = "Fiiish! RS"
+description = "A simple remake of the Fiiish endless swimmer in Rust."
+categories = ["action", "released"]
```

Comment:
I'm not sure the project belongs to the "released" category atm:

> Finished games that have been released

The game's page says it's "still in early development."

Reply:
The games page is wrong. 🤦  Will update once I get out of this train.
The rust version of the game has parity with the iOS version.

Reply:
:ok_hand: